aws_docker_creds.sh

```diff
@@ -13,6 +13,25 @@ cat << EOF > ~/.aws/config
 region = $AWS_REGION
 EOF
 
+# For multi account aws setups, use primary credentials to assume the role in
+# the target account
+if [[ -n $AWS_STS_ROLE || -n $AWS_STS_ACCOUNT ]]; then
```

Comment:
Do you think this should be before we have the expectation for AWS_ACCESS_KEY_ID and other variables? That way a user could set STS instead of the other values, rather than override them